pvl/__init__.py

```diff
@@ -60,20 +62,27 @@ def load(path, parser=None, grammar=None, decoder=None, **kwargs):
 
 def get_text_from(path) -> str:
     try:
-        p = Path(path)
-        return p.read_text()
+        if isinstance(path, str) and path.startswith('http'):
+            with urllib.request.urlopen(path) as f:
+                c = f.read().decode('utf-8').replace('\r', '')
```

Comment:
Why are you stripping "\r" characters here?  They are allowable characters in PVL text and should be left as-is, as far as I can tell.

Reply:
this was a bad attempt to fix what I thought was the issue with that cassini image until I read the error carefully, can remove.